e2e/test/scenarios/dashboard-filters/dashboard-filter-defaults.cy.spec.ts

```diff
@@ -107,10 +108,12 @@ describe("scenarios > dashboard > filters > reset", () => {
     filterWidget().contains("Filter One").should("be.visible");
     filterWidget().contains("Foo").should("be.visible");
 
+    cy.get<number>("@dashboardId").then(dashboardId =>
+      visitDashboard(dashboardId),
```

Comment:
I've tried a lot of solutions that are less drastic but this is the only one that works for me.